prometheus/rootfs/bin/boot.sh

```diff
@@ -22,18 +22,23 @@ done
 
 set +e
 
-OPTS=""
+until [ -n "${DEIS_MONITOR_ALERT_SERVICE_HOST+1}" ]; do
```

Comment:
i thought this is how service information was exposed to nodes.  the variable gets set when the service is created.